test/test_jit.py

```diff
@@ -1309,7 +1309,8 @@ def forward(self, x):
         torch._C._jit_pass_constant_propagation(scriptM.graph)
         # TODO: Build the qparam_dict from parse_ir directly for this pass
         qparam_dict = _helper_generate_qparam(scriptM, input_data)
-        torch._C._jit_pass_insert_quantdequant(scriptM.graph, qparam_dict)
+        torch._C._jit_pass_insert_quantdequant(scriptM._c, "forward",
```

Comment:
is this a public API? it starts with underscore, does this mean it might be changed in the future?

Reply:
could you add a comment here if they are expected to change in the future and what should be the correct API that we want to use?

Reply:
._c is used to access the moduleObj for the c++ native code. This is how its exposed by JIT as i spoke to them last time.